src/app/3d/qgs3dmapconfigwidget.cpp

```diff
@@ -252,6 +252,11 @@ Qgs3DMapConfigWidget::Qgs3DMapConfigWidget( Qgs3DMapSettings *map, QgsMapCanvas
   groupExtent->setCurrentExtent( mMap->extent(), mMap->crs() );
   groupExtent->setOutputExtentFromCurrent();
   groupExtent->setMapCanvas( mMainCanvas );
+
+  // checkbox to display the extent in the 2D Map View
+  mShowExtentIn2DViewCheckbox = new QCheckBox( tr( "Show in 2D Map View" ) );
```

Comment:
```suggestion
  mShowExtentIn2DViewCheckbox = new QCheckBox( tr( "Show in 2D map view" ) );
```

Reply:
Done. I have also added an other commit to change it for some other labels.